### PR TITLE
- Support sandboxing on FreeBSD via capsicum (from 11.x intentionally

### PR DESCRIPTION
**Pull request by devnexen**
even though exists from 10.x but did not work as expected).
- Fixes arc4random environment variable "propagation".

### CODE REVIEW

**Comment by radare** [comment]
w t f
screen shot 2018-06-12 at 16 47 13

File: libr/util/sandbox.c [hunk 1]
```diff
@@ -112,6 +122,12 @@ R_API bool r_sandbox_enable (bool e) {
 		eprintf ("sandbox: pledge call failed\n");
 		return false;
 	}
+#endif
+#if HAVE_LIB_CASPER
+	if (enabled && cap_enter() != 0) {
```
Comment (radare) [inline libr/util/sandbox.c]: space before (

**Comment by devnexen** [comment]
I think I know why the regression fails. This is a change I planned for later (for the sandboxing) but finally I have to do it now.

**Comment by devnexen** [comment]
Will fix in another PR, creating new one just to fix arc4random.

**Pull request closed without merging**